cmake/onnxruntime_kernel_explorer.cmake

```diff
@@ -64,7 +64,9 @@ elseif (onnxruntime_USE_ROCM)
     # Currently we shall not use composablekernels::device_operations, the target includes all conv dependencies, which
     # are extremely slow to compile. Instead, we only link all gemm related objects. See the following link on updating.
     # https://github.com/ROCmSoftwarePlatform/composable_kernel/blob/85978e0201/library/src/tensor_operation_instance/gpu/CMakeLists.txt#L33-L54
-    device_gemm_instance)
+    device_gemm_instance
+    device_gemm_add_fastgelu_instance
+    device_gemm_fastgelu_instance)
```

Comment:
If the symbols present in target `onnxruntime_providers_rocm`, then I do believe we can remove all these together. Can you try it out, and move the comment to `onnxruntime_providers.cmake`

Reply:
done.